tools/ffmpeg/merge-feature-branches.sh

```diff
@@ -2,16 +2,15 @@
 
 # useage: merge-feature-branches [--rpi]
 
-merge() {
-  git fetch $1 $2
-  git merge --no-ff --log FETCH_HEAD
+pull() {
+  git pull --no-ff --log "$@"
 }
 
-merge https://github.com/xbmc/FFmpeg.git 4.3-Matrix-Alpha1
-merge https://github.com/lrusak/FFmpeg.git v4l2-drmprime-v5
-merge https://github.com/Kwiboo/FFmpeg.git v4l2-request-hwaccel-4.3
-merge https://github.com/LibreELEC/FFmpeg.git 4.3-libreelec-misc
+pull https://github.com/xbmc/FFmpeg.git 4.3-Matrix-Alpha1
+pull https://github.com/lrusak/FFmpeg.git v4l2-drmprime-v5
+pull https://github.com/Kwiboo/FFmpeg.git v4l2-request-hwaccel-4.3
```

Comment:
You should skip this for `--rpi` or there may be conflicts when pulling in the `rpi` branch as they can become out-of-sync.

I will push support for 4:2:2 and 10-bit formats any day now and that will cause conflicts with changes from `rpi` branch until that branch is updated.

Ideally we can incorporate the rpi changes directly into the v4l2-request-hwaccel work but the rpi hevc threading part needs to be reworked before that can happen.

Reply:
Yes, that can be dropped for `--rpi` - I mainly kept that in to reflect current state of branches